web/react/components/post_list.jsx

```diff
@@ -439,9 +439,9 @@ module.exports = React.createClass({
                 currentPostDay = utils.getDateForUnixTicks(post.create_at);
                 if (currentPostDay.toDateString() != previousPostDay.toDateString()) {
                     postCtls.push(
-                        <div className="date-separator">
-                            <hr className="separator__hr" />
-                            <div className="separator__text">{currentPostDay.toDateString()}</div>
+                        <div key="date_div" className="date-separator">
```

Comment:
it's not uniqe key when date separators are more then one. I shoud use something: `post.create_at` or  `currentPostDay.toDateString()`. And of course keys for all elements are not needed, only for dynamic children nodes